lib/master.go

```diff
@@ -408,6 +434,22 @@ func (m *DefaultBaseMaster) registerHandlerForWorker(ctx context.Context, worker
 	return nil
 }
 
+// generateWorkerID assigns a new worker id.
+// When creating a job master, job manager provides a pre allocated ID, in other
+// cases, we need to generate a random WorkerID.
+func (m *DefaultBaseMaster) generateWorkerID(workerType WorkerType, config WorkerConfig) string {
```

Comment:
As to the. jobmaster , job name  is more reasonable .

Reply:
It is used in two scenarios,
- job manager generates job master id
- job master generates worker id

Using worker id is more generic.

Reply:
ptal @nongfushanquan

Reply:
As for using `job name` as worker id of job master, added a TODO and refine it later. Since the `job name` should be checked unique.